src/ciceroscm/carbon_cycle/carmen_wrapper.py

```diff
@@ -0,0 +1,169 @@
+        "gpp_c_tan2": 0.0,
+        "lit_t_l": 1.143e-05,
+        "lit_t_e": 0.01792216,
+        "lit_c_l": -0.61640124,
+        "lit_c_half": 256.69192703,
+        "lit_c_e": -0.23731487,
+        "lit_hyst": 0.0167247,
+        "lit_c_tan": 0.0,
+        "lit_fast": 0.92847622,
+        "lit_slow": 0.91882796,
+        "lit_c_tan2": 0.0,
+        "vres_t_l": 8.85e-06,
+        "vres_t_e": -0.07693331,
+        "vres_c_l": -0.47443673,
+        "vres_c_half": 808.47364366,
+        "vres_c_e": -0.37496243,
+        "vres_hyst": -0.04767878,
+        "vres_c_tan": 0.0,
+        "vres_fast": 0.90821573,
+        "vres_slow": 0.84546906,
+        "vres_c_tan2": 0.0,
+        "sres_t_l": 8.76e-06,
+        "sres_t_e": 0.06184566,
+        "sres_c_l": 1.43982004,
+        "sres_c_half": 92.12981868,
+        "sres_c_e": -0.24272656,
+        "sres_hyst": 0.04094068,
+        "sres_c_tan": 0.0,
+        "sres_fast": 0.89815371,
+        "sres_slow": 0.85285196,
+        "sres_c_tan2": 0.0,
+        "npp_t_l": 0.20958114,
+        "npp_t_e": -0.09892429,
+        "npp_c_l": -0.19742283,
+        "npp_c_half": 281.66916948,
+        "npp_c_e": 0.05305683,
+        "npp_hyst": 0.07758387,
+        "npp_c_tan": 0.0,
+        "npp_fast": 0.94904171,
+        "npp_slow": 0.71402205,
+        "npp_c_tan2": 0.0,
+        "docn": 41.80816876,
+        "docnfac": 9.50761817,
+        "ocntemp": 0.0, 
+        "docntemp": 0.12145903,
+    }
+    carbon_cycle_model_required_pamset ={}
+
+    def __init__(self, pamset_emiconc, pamset_carbon=None):
+        """
+        Initialize the Carbon Cycle Model.
+
+        Parameters
+        ----------
+        pamset_emiconc : dict
+            Parameter set from the concentrations emission handler, containing:
+            - idtm: Number of subyearly timesteps (e.g., 24 for monthly steps).
+            - nystart: Start year of the simulation.
+            - nyend: End year of the simulation.
+        pamset_carbon : dict
+            Optional carbon specific parameter set
+            TODO: explain what the model is expecting
+        """
+        # Pass the necessary variable to the parent class
+        super().__init__(pamset_emiconc, pamset_carbon=pamset_carbon)
+
+        if pamset_carbon is None:
+            pamset_carbon = self.carbon_cycle_model_required_pamset
+
+        # Create and store an object implementing CARMEN
+        carmen_instance = CarbonCycle(
+            # if you want to use ESM scenario data
+            # {"model": "NorESM2-LM", "scenario": "ssp126"},
+            # if you want to run without any pre-loaded scenario
+            {"model": pamset_carbon.get("ESM_to_emulate", "UKESM1-0-LL"), "initial_year": pamset_emiconc["nystart"], "final_year": pamset_emiconc["nyend"]},
+            1/pamset_emiconc["idtm"],
+            1/pamset_emiconc["idtm"],
+            npp_flag=True,
+            **pamset_carbon,
+        )
+        self.carmen = carmen_instance
+

```

Comment:
However, you do need to overwrite the super function for reset_co2_hold to allow for multiple runs with the same instance. Both so it can be called to change the ESM_to_emulate for instance in there (for parallel runs), and also to reset pools and year-index as the abstract method doesn't know what Carmen needs for this...